content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -7,7 +7,8 @@ content_type: concept
 weight: 30
 ---
 <!-- overview -->
-This command initializes a Kubernetes worker node and joins it to the cluster.
+This command initializes a Kubernetes node and joins it to the cluster.
```

Comment:
WDYT?

```suggestion
This command initializes a new Kubernetes node and joins it to the existing cluster.
```

Reply:
yes please use this exactly
https://github.com/kubernetes/website/issues/48399#issuecomment-2418771122

Reply:
Done.